ads/google/a4a/utils.js

```diff
@@ -414,13 +417,20 @@ export function extractAmpAnalyticsConfig(a4a, responseHeaders) {
     const slotId = a4a.element.getAttribute('data-amp-slot-index');
     const qqid = (responseHeaders && responseHeaders.has(QQID_HEADER))
         ? responseHeaders.get(QQID_HEADER) : 'null';
-    config['requests']['visibilityCsi'] =
-        'https://csi.gstatic.com/csi?fromAnalytics=1' +
-        `&c=${correlator}&slotId=${slotId}&qqid.0=${qqid}`;
+    const baseCsiUrl = 'https://csi.gstatic.com/csi?s=a4a' +
+        `&c=${correlator}&slotId=${slotId}&qqid.${slotId}=${qqid}`;
+    const now = a4a.win.performance && a4a.win.performance.now ?
+        a4a.win.performance.now() : Date.now();
+    opt_initTime = opt_initTime || 0;
+    const time = Math.round(now - opt_initTime);
```

Comment:
Please name `delta_time` or `delta` or something so that it's clear that you're not talking about an absolute time here.